src/main/target/NAZE/initialisation.c

```diff
@@ -19,27 +19,24 @@
 #include <stdint.h>
 
 #include "platform.h"
+#include "drivers/bus.h"
 #include "drivers/bus_i2c.h"
 #include "drivers/bus_spi.h"
 #include "io/serial.h"
 #include "hardware_revision.h"
 
-void targetBusInit(void)
```

Comment:
Ah, it's a remains from `targetBusInit` declared as `targetBusInit(i2cConfig_t *)` to call `i2cHardwareConfigure`; the `i2cHardwareConfigure` now does not take `i2cConfig_t *`, and I forgot to put `void` back in place. I'll fix it.